cylc/flow/rundb.py

```diff
@@ -639,19 +639,18 @@ def select_task_job(self, cycle, name, submit_num=None):
     def select_jobs_for_restart(self, callback):
         """Select from task_pool+task_states+task_jobs for restart.
 
-        Invoke callback(row_idx, row) on each row, where each row contains:
-            [cycle, name, status, submit_num, time_submit, time_run,
-             time_run_exit, job_runner_name, job_id, platform_name]
+        Invoke callback(row_idx, row) on each row of the result.
         """
```

Comment:
Uh-oh, these changes are going to conflict with https://github.com/cylc/cylc-flow/pull/4906#pullrequestreview-1003719062